src/main/scala/ensime/consierge/IssueFetching.scala

```diff
@@ -66,7 +50,14 @@ trait IssueFetching {
       response.entity
         .toStrict(config.timeout)
         .map { entity =>
-          Json.parse(entity.data.decodeString("UTF-8")).as[Seq[Issue]]
+          val dt = entity.data.decodeString("UTF-8")
+          try {
+            Json.parse(dt).as[Seq[Issue]]
+          } catch {
+            case e: Exception =>
+              logger.error(s"Unable to parse server results: $dt")
+              throw e
```

Comment:
Agree, but this allows to see original json (without logging it all the time). That shows me github limits problem.